modules/builtin/src/bot-templates/welcome-bot/qna/xr9uskkhv0_forget_my_name.json

```diff
@@ -7,7 +7,7 @@
     "questions": {
       "en": [
         "forget my name",
-        "don't remember me",
+        "dont remember me",
```

Comment:
I guess this change is not intentional.

Reply:
Good point, I've copy-pasted files and something happened, shouldn't be